### PR TITLE
Make bundle-wheel.sh more configurable

### PR DESCRIPTION
**Pull request by klecki**
## Description
- [ ] **Bug fix** (*non-breaking change which fixes an issue*)
- [ ] **New feature** (*non-breaking change which adds functionality*)
- [ ] **Breaking change** (*fix or feature that would cause existing functionality to not work as expected*)
- [ ] **Refactoring** (*Redesign of existing code that doesn't affect functionality*)
- [x] **Other** (*e.g. Documentation, Tests, Configuration*)

### What happened in this PR
Allow to configure output path for bundle wheel script.
Add option to not compress the output wheel.
Those changes should allow for easier and faster development work.
Removing compression saves about a minute of wheel packaging.

#### Additional information
- Affected modules and functionalities:

- Key points relevant for the review:

### Tests
- [x] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by klecki** [comment]
!build

File: dali/python/bundle-wheel.sh [hunk 1]
```diff
@@ -48,7 +48,15 @@ STRIP_DEBUG=${2:-NO}
 TEST_BUNDLED_LIBS=${3:-NO}
 OUTWHLNAME=${4:-$(basename $INWHL)}
 DEPS_PATH=${5:-/usr/local}
-OUTDIR=/wheelhouse
+OUTDIR=${6:-/wheelhouse}
```
Comment (JanuszL) [inline dali/python/bundle-wheel.sh:51]: Can you adjust build_helper.sh, build.sh, Dockerfile, and Dockerfile.build.aarch64-linux to allow passing these variables from the outside so even the local build using docker environment could benefit from it?

Comment (klecki) [reply dali/python/bundle-wheel.sh:51]: I tried adding it, compression might be useful, the target directory for docker would require handling the in-docker and out of docker targets probably, so I just left it as is.

**Review by JanuszL — APPROVED** [review]

File: dali/python/bundle-wheel.sh (hunk 1 above)
Comment (JanuszL) [reply dali/python/bundle-wheel.sh:51]: Looks good now.

**Comment by klecki** [comment]
!build

**Review by jantonguirao — APPROVED** [review]

**Pull request merged**